.releaserc.js

```diff
@@ -2,6 +2,7 @@ module.exports = {
     branches: [
         {name: 'main'},
         {name: '1.87.x', range: '1.87.x', channel: '1.87.x'},
+        {name: 'ng-15', prerelease: 'beta'},
```

Comment:
dont we need to remove this before merging?

Reply:
removed it